addon/-decorators.js

```diff
@@ -1,7 +1,7 @@
 let modifierNames = ['restartable', 'drop', 'enqueue', 'maxConcurrency', 'cancelOn'];
 let decorators = {};
 
-function makeDecorator(modifierName, methodName) {
+export function makeDecorator(modifierName, methodName) {
```

Comment:
Why is this needed?

fwiw "decorators" (as they're called here) never really caught on and have been removed from the `next` branch. Either way you shouldn't need this for this PR.